### ISSUE TITLE
store the graph hashes as extra

### ISSUE THREAD

**Issue by kjappelbaum** [danieleongari/CURATED-COFs#21]
the "efficient" duplicate filtering i do uses the Weisfeiler-Lehman graph hash + the composition. This is something you could just store somewhere as extra and then use for filtering before you make a release (you can get this from the `MOFChecker` package).

**Comment by kjappelbaum** [comment]
This "cheap" check on the initial structures from master
[initial_structs.txt](https://github.com/danieleongari/CURATED-COFs/files/5879794/initial_structs.txt)

**Comment by kjappelbaum** [comment]
here for the optimized ones

[optimized_structs.txt](https://github.com/danieleongari/CURATED-COFs/files/5882662/optimized_structs.txt)

**Comment by danieleongari** [comment]
Thanks @kjappelbaum ,
I checked all the structures (https://docs.google.com/document/d/1h5mAFV5ZUN6JuT_qsBv1pxRWa5NnSrm9CRzFsaeUy8w/edit?usp=sharing)
and the method seems to be pretty reliable. 
Still it is not clear to me why some exact matches got recognized only after the optimization!

How we can reproduce these results? We might want to include the hashcode as a column in `cof-frameworks.csv` more than as an extra, because it is the stage when we want to doublecheck and discard structures from the database.

**Comment by kjappelbaum** [comment]
The difference between the optimized and initial ones is due to the method we use to get the graphs. It might not find the bonds between the atoms in the unoptimized structures. If you can point out some examples i will give them a specific look and maybe change the defaults. 

To reproduce this, you basically only need to use https://github.com/kjappelbaum/mofchecker (deployed at https://mofchecker.matcloud.xyz/). There are still a range of improvements I need to make for this package but it might also make sense if one can already use some of the checks to just refine them.

**Comment by ltalirz** [comment]
@danieleongari could you provide a folder with the structures that are only recognized after optimization?

**Comment by danieleongari** [comment]
Since this requires a bit more investigation, I propose to postpone this to the next update, since the Feb21 one is ready to merge

**Comment by danieleongari** [comment]
Seen that the graph-hash is still ongoing development, we include it as manual check to compute on-the-fly: https://github.com/danieleongari/CURATED-COFs/pull/27
